components/sup/src/manager/service_updater.rs

```diff
@@ -311,7 +313,23 @@ impl Periodic for Worker {
         match env::var(FREQUENCY_ENVVAR) {
             Ok(val) => {
                 match val.parse::<i64>() {
-                    Ok(num) => num,
+                    Ok(num) => {
+                        if env::var(FREQUENCY_BYPASS_CHECK_ENVVAR).is_ok() ||
+                            num >= MIN_ALLOWED_FREQUENCY
+                        {
+                            num
+                        } else {
+                            outputln!(
+                                "{} value ({}) is below the minimal authorized value ({}) \
+                                Falling back to minimal {} MS frequency.",
+                                FREQUENCY_ENVVAR,
+                                num,
+                                MIN_ALLOWED_FREQUENCY,
+                                MIN_ALLOWED_FREQUENCY
+                            );
+                            MIN_ALLOWED_FREQUENCY
+                        }
+                    }
```

Comment:
We should probably handle the (hopefully rare!) case when someone sets a value to 0 or less with the bypass in place.

Reply:
Added a different message in case it's negative ;)